docs/src/markdown/EmptyState.mdx

```diff
@@ -0,0 +1,90 @@
+import Box from '@mui/material/Box';
+import EmptyStateImage from './images/emptyState.png';
+
+# Empty State
+
+The `<EmptyState>` component is an element that can be used as a placeholder when no data is present (such as an empty list, or a placeholder page for future content). This is only used when no data is available, rather than during loading (see [empty states pattern](https://brightlayer-ui.github.io/patterns/empty-states)).
+
+<Box
+    sx={{
+        width: '100%',
+        textAlign: 'center',
+    }}
+>
+    <Box
+        sx={{
+            width: '100%',
+            maxWidth: '200px',
+        }}
+        component="img"
+        alt="Empty State Image"
+        src={EmptyStateImage}
+    />
+</Box>
+
+## Empty State API
+
+<Box
```

Comment:
As markdownMapping has been merged to base branch, which will also cover styling  for tables in mdx, so we wont need sx prop here. Just keeping Box around the table works fine.

Reply:
so all this isn't needed now?

<img width="301" alt="image" src="https://user-images.githubusercontent.com/37150565/183641536-728d6cd6-7131-42f9-9409-330fb2b9311e.png">

Reply:
yep